modules/packages/LinearAlgebra.chpl

```diff
@@ -823,6 +825,11 @@ proc matPow(A: [], b) where isNumeric(b) {
   if !isSquare(A) then
     halt("Array not square");
 
+  if (usingLAPACK && b < 0) {
```

Comment:
We typically define a separate overload for `usingLAPACK` variants to handle the `isBLASType` check. For example, the `LAPACK` version would not work on an array of `int(16)`.